src/bot/commands/info.rs

```diff
@@ -61,8 +62,14 @@ impl Info {
     }
 }
 
+#[async_trait]
 impl Command for Info {
-    fn execute(&self, db_pool: Pool<ConnectionManager<PgConnection>>, api: Api, message: Message) {
+    async fn execute(
```

Comment:
this function is called in each command's impl
```rust
    pub fn execute(db_pool: Pool<ConnectionManager<PgConnection>>, api: Api, message: Message) {
        Self {}.execute(db_pool, api, message);
    }
```
since you changed trait methods to async, you should await

```rust
    pub async fn execute(db_pool: Pool<ConnectionManager<PgConnection>>, api: Api, message: Message) {
        Self {}.execute(db_pool, api, message).await;
    }
```